vnext/ReactUWP/Utils/Helpers.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <stdint.h>
+#include <winrt/Windows.UI.Xaml.h>
+#include <IReactInstance.h>
+#include <folly/dynamic.h>
+
+namespace winrt {
+  using namespace Windows::UI::Xaml;
+}
+
+namespace react { namespace uwp {
+
+  using namespace std;
+
+  struct ReactId {
+    int64_t tag{ 0 };
+    bool isValid{ false };
+  };
+
+  template<typename T>
+  inline typename T as(folly::dynamic const& obj)
+  {
+    return (T)(obj.asInt());
+  }
+
+  template<>
+  inline typename std::string as<std::string>(folly::dynamic const& obj)
```

Comment:
not sure how useful the wrappers are in general, we're letting people write as<boo> instead of asBool() . ?

also if we need helpers like this we have ValueUtils.h

Reply:
I suggest we integrate them to as. Personally I think as<std::wstring>/as<winrt::hstring> is more readable than asWStr/asHstring

Reply:
aswstr/ashstring exist as separate functions because a conversion from std::string to wstring/hstring needs to happen.  Sure the names could be a little different if you prefer.. I try not to even use aswstr because there is reactwindowscore\unicode.h for that, really only need ashstring

Your wrappers here are not doing any conversions just changing asString() to as<std::string>(), I'm not sure the difference is worth it, seems confusing as it is just renaming a function that we use heavily.  What is probably more interesting with the bulk of the places that use asstring/asdouble/asbool is if we had a set of utilities like RCTConvert in ios that hides most of our dynamic handling.  BUT with fabric coming and removing dynamic all together maybe not worth the effort

Reply:
I removed it and no follow up item.